inlineplz/interfaces/github.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, owner, repo, pr, token, url=None):
             self.github = github3.GitHub(token=token)
         else:
             self.github = github3.GitHubEnterprise(url, token=token)
+        self.user = self.github.me()
```

Comment:
this seems dangerous because if you reuse your own personal account for running inline-plz, we could end up deleting legit non-inline-plz comments. maybe add some sort of inline-plz token to every comment that we can then scan for?